Sources/StreamChatUI/ChatMessageList/ChatChannelVC.swift

```diff
@@ -99,4 +99,27 @@ extension _ChatChannelVC: _ChatChannelControllerDelegate {
     ) {
         messageList.updateMessages(with: changes)
     }
+
+    public func channelController(
+        _ channelController: _ChatChannelController<ExtraData>,
+        didChangeTypingMembers typingMembers: Set<_ChatChannelMember<ExtraData.User>>
+    ) {
+        guard let currentMember = channelController.channel?.membership else {
+            typingIndicatorView.setAnimatedly(hidden: true)
+            log.error("Current member not found in ChatChannelVC.")
+            return
+        }
```

Comment:
I wouldn't check for a membership here. Users don't have to be members to create typing events. Better to check for the current logged in user directly.

Reply:
Thanks, fixed)